creativecloud/blocks/catalog/catalog.js

```diff
@@ -25,12 +25,28 @@ export default async function init(el) {
       const { default: initSidenav } = await import('../sidenav/sidenav.js');
       const sidenav = await initSidenav(sidenavEl);
       el.appendChild(sidenav);
+      sidenav.setAttribute('daa-lh', 'b1|catalog-sidenav');
```

Comment:
b1 is usually the marquee though.
should we assign the number of block manually like this?

Reply:
bX is the number within the section, and this one is (and will be) the first always per catalog component layout